packages/minisign/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://jedisct1.github.io/minisign/
+TERMUX_PKG_DESCRIPTION="A dead simple tool to sign files and verify signatures"
+TERMUX_PKG_LICENSE="ISC"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.9
+TERMUX_PKG_DEPENDS="libsodium"
+TERMUX_PKG_BUILD_DEPENDS="pkg-config"
+TERMUX_PKG_SRCURL=https://github.com/jedisct1/minisign/archive/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=caa4b3dd314e065c6f387b2713f7603673e39a8a0b1a76f96ef6c9a5b845da0f
```

Comment:
Please add a newline at the end, to ensure that scripts can correctly parse the SHA256.